Mathlib/AlgebraicGeometry/AffineScheme.lean

```diff
@@ -216,6 +216,18 @@ theorem isBasis_affine_open (X : Scheme) : Opens.IsBasis X.affineOpens := by
   exact rangeIsAffineOpenOfOpenImmersion _
 #align algebraic_geometry.is_basis_affine_open AlgebraicGeometry.isBasis_affine_open
 
+theorem iSup_affineOpens_eq_top (X : Scheme) : ⨆ i : X.affineOpens, (i: Opens X) = ⊤ := by
```

Comment:
`TopologicalSpace.IsTopologicalBasis.sUnion_eq` and `Opens.ext` should get you almost there.